_posts/en/2023-10-06-hatsumi-taikai.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 title:  "Hatsumi Taikai"
 date:   2023-10-06 16:00:00 +0200
-last_modified_at: 2023-10-07 9:00:00 +0200
```

Comment:
Please add "last modified" text next to the Player table, too - so that people won't ask us for an update 5 times a day

Reply:
last modified date is included directly below the table as well but fair enough :taco: